modules/hal_nordic/nrfx/Kconfig

```diff
@@ -9,6 +9,10 @@ menu "nrfx drivers"
 
 rsource "Kconfig.logging"
 
+config MPSL
+	bool
+	# Dummy kconfig to allow compiling outside of NCS
```

Comment:
It would be better to locate this dummy option in `modules/hal_nordic/Kconfig` as it's not related to nrfx only.

Reply:
done